### DISCUSSION TITLE
Build a particular commit

### DISCUSSION THREAD

**Discussion by mynzocarbon** [codemagic-ci-cd/codemagic-docs#1782] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

React Native

### Steps to reproduce

1. Select a previous successful build
2. Click on little play icon
3. Select "Build commit #24xxx"
4. Click "Start new build"

and yet it still build on the latest commit.

### Expected results

Build on the commit ID selected.

### Actual results

Builds using the latest commit.

### Build id (optional)

648c5a4f05f120857f9aedcc

**Comment by dtrdic** [comment]
hello @mynzocarbon, I can reproduce it, and it looks like there are some issues. I'll update you as soon as it's fixed.

**Reply by dtrdic** [reply]
@mynzocarbon this issue should be fixed now, can you please confirm?
